cpu/em_quad_level_frequency_transitions.py

```diff
@@ -61,6 +62,19 @@ def setUp(self):
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
 
+        fre_min = 0
+        fre_max = 0
+        freq_info = process.system_output(
+            "cpupower frequency-info", shell=True).decode("utf-8")
+        for line in str(freq_info).splitlines():
+            if re.search('hardware limits:', line, re.IGNORECASE):
```

Comment:
are we looking at one of multiple such matching lines ?

Reply:
Also, do we really need `re` here ? `if 'hardware limits:' in line:` won't work ?